### DISCUSSION TITLE
What is the state of Metal bindings?

### DISCUSSION THREAD

**Discussion by anose001** [dotnet/Silk.NET#2321] [Q&A] [answered]
I wasnt able to find proper information for what the current state for Metal bindings for the apple ecosystem is, especially for iOS and macOS in the context of compute operations. 

As an alternative is Vulkan via MoltenVK on iOS or macOS supported in the above context?

**Comment by Perksey** [comment, marked as answer]
We haven’t started on Metal bindings yet. MoltenVK is supported.
